conda/recipes/librmm/meta.yaml

```diff
@@ -65,9 +61,7 @@ outputs:
         - cuda-version ={{ cuda_version }}
       run:
         - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
-        {% if cuda_major == "11" %}
-        - cudatoolkit
-        {% endif %}
+        - {{ "cudatoolkit" if cuda_major == "11" else "cuda-version" }}
```

Comment:
The "else" isn't really necessary here unless it's required for syntactical reasons. We could leave the "else" clause empty, we don't need `cuda-version` again since it's already given by `pin_compatible('cuda-version', ...)` one line above this.